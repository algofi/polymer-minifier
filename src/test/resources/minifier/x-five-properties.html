
<dom-module id="x-five-properties">
	<template>
		<style>
		</style>
		{{sessionId}} [[friends]]
		
		[[posts]] [[habilitation.id]]
		
		{{userId.name}}
		<script>
			Polymer( {
				is:'x-five-properties',
				properties: {
					sessionId: Number,
					userId: {
						type: String,
						notify: true,
						readOnly: true,
						observer: '_userIdChanged'
					},
					habilitation: {
						type: Object
					},
					friends: {
						type: Array
					},
					posts: {
						type: Object
					}
				},
				_userIdChanged: function() {
					console.log( this.sessionId, this.userId, this.habilitation.id, this.friends.list, this.posts );
				},
				ready: function () {
					var self = this;
					this.addEventListener( 'tap', function ( e ) {
						console.log( self.sessionId, self.userId, self.habilitation.id, self.friends.list, self.posts );
					} );
				},
				created: function () {
					var me = this;
					this.addEventListener( 'tap', function ( e ) {
						console.log( me.sessionId, me.userId, me.habilitation.id, me.friends.list, me.posts );
					} );
				}
			} );
			
		</script>
	</template>
</dom-module>