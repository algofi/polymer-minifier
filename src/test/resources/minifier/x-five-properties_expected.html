<dom-module id="x-five-properties">
	<template>
		<style>
		</style>
		{{a}} [[d]] 
		
		[[e]] [[c.id]]
		
		{{b.name}}
		<script>
			Polymer( {
				is:'x-five-properties',
				properties: {
					a: Number,
					b: {
						type: String,
						notify: true,
						readOnly: true,
						observer: '_userIdChanged'
					},
					c: {
						type: Object
					},
					d: {
						type: Array
					},
					e: {
						type: Object
					}
				},
				_userIdChanged: function() {
					console.log( this.a, this.b, this.c.id, this.d.list, this.e );
				},
				ready: function () {
					var self = this;
					this.addEventListener( 'tap', function ( e ) {
						console.log( self.a, self.b, self.c.id, self.d.list, self.e );
					} );
				},
				created: function () {
					var me = this;
					this.addEventListener( 'tap', function ( e ) {
						console.log( me.a, me.b, me.c.id, me.d.list, me.e );
					} );
				}
			} );
			
		</script>
	</template>
</dom-module>